snomed/com.b2international.snowowl.validation.snomed/src/main/resources/scripts/ruleSnomedCommon4.groovy

```diff
@@ -35,9 +41,33 @@ SnomedConceptSearchRequestBuilder conceptsRequestBuilder = SnomedRequests.prepar
 		.setExpand("preferredDescriptions()")
 				
 if (params.isUnpublishedOnly) {
-	conceptsRequestBuilder.filterByEffectiveTime(EffectiveTimes.UNSET_EFFECTIVE_TIME)
+
+	def unpublishedLangueMembers = SnomedRequests.prepareSearchMember()
```

Comment:
Rename `unpublishedLangueMembers` to `descriptionsIdsWithUnpublishedLanguageMembers`.